metadata.json

```diff
@@ -43,7 +43,8 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8"
+        "8",
+        "9"
```

Comment:
Hi. @markgraf .
Can you also enable acceptance tests for debian ?

Reply:
Took me a while to smarten myself up a bit :-)
Next commit should do the trick.  I have no AWS-Account for testing though. :-/